internal/provider/provider_test.go

```diff
@@ -62,11 +64,19 @@ func setupVCR(t *testing.T, cassetteName string) *recorder.Recorder {
 	hookRedactIntegrationUserCredentials := func(i *cassette.Interaction) error {
 		intUser := os.Getenv("BTP_USERNAME")
 		intUserPwd := os.Getenv("BTP_PASSWORD")
+		intUserIdp := os.Getenv("BTP_IDP")
 
 		firstName, lastName := getNameFromEmail(intUser)
 
 		if strings.Contains(i.Request.URL, "/login/") {
 			i.Request.Body = strings.ReplaceAll(i.Request.Body, intUserPwd, "redacted")
+			i.Request.Body = strings.ReplaceAll(i.Request.Body, "\"customIdp\":\""+intUserIdp+"\"", "\"customIdp\":\"\"")
+			i.Response.Body = strings.ReplaceAll(i.Response.Body, "\"issuer\":\""+intUserIdp+"\"", "\"issuer\":\"accounts.sap.com\"")
+			i.Response.Body = strings.ReplaceAll(i.Response.Body, "\"issuer\":\""+intUserIdp+".accounts400.ondemand.com\"", "\"issuer\":\"accounts.sap.com\"")
+		}
+
+		if _, exists := i.Request.Headers[http.CanonicalHeaderKey(btpcli.HeaderCLICustomIDP)]; exists {
```

Comment:
```suggestion
		if _, exists := i.Request.Headers[btpcli.HeaderCLICustomIDP]; exists {
```

In favour of readability I would prefer to modify the constants in the btpcli package, such that they are canonical by default.

Reply:
agree. I was not sure if it was safe to change the constant. Won't this brake the already recorded fixtures?

Reply:
Done